### PR TITLE
Add API for adding external storage account configurations to a web app

### PR DESCRIPTION
**Pull request by ebencarek**
Redo of #3452, which was merged accidentally. The API is not live yet.

This checklist is used to make sure that common issues in a pull request are addressed. This will expedite the process of getting your pull request merged and avoid extra work on your part to fix issues discovered during the review process.

### PR information
- [x] The title of the PR is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).
- [x] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [x] If applicable, the PR references the bug/issue that it fixes.
- [x] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [x] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [x] My spec meets the review criteria:
  - [x] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [x] The spec follows the guidelines described in the [Swagger checklist](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md) document.
  - [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

File: specification/web/resource-manager/Microsoft.Web/stable/2018-02-01/CommonDefinitions.json [hunk 1]
```diff
@@ -507,6 +507,53 @@
         }
       }
     },
+    "AzureStorageInfoValue": {
+      "description": "Azure Files or Blob Storage access information value for dictionary storage.",
+      "type": "object",
+      "properties": {
```
Comment (dsgouda) [inline specification/web/resource-manager/Microsoft.Web/stable/2018-02-01/CommonDefinitions.json]: nit: Please check if any of these properties are readOnly

**Review by dsgouda — CHANGES_REQUESTED** [review]
LGTM for the most part, please address the comment

File: specification/web/resource-manager/Microsoft.Web/stable/2018-02-01/CommonDefinitions.json (hunk 1 above)
Comment (ebencarek) [reply specification/web/resource-manager/Microsoft.Web/stable/2018-02-01/CommonDefinitions.json]: State should be readOnly, fixed in latest commit.

**Review by dsgouda — APPROVED** [review]
LGTM

**Comment by dsgouda** [comment]
@ebencarek Let me know when this is ready to merge

**Comment by dsgouda** [comment]
@ebencarek Please let us know when this is ready to be merged. Will close this week if there is no activity on the PR.

**Comment by ebencarek** [comment]
@dsgouda This should be ready to merge on Friday.

**Comment by dsgouda** [comment]
Please post on the PR once ready.

**Comment by dsgouda** [comment]
@ebencarek gentle almost-Friday ping

**Comment by ebencarek** [comment]
@dsgouda Go ahead and merge. Thanks!

**Comment by dsgouda** [comment]
@azuresdkci Retest this please

**Comment by lmazuel** [comment]
@AutorestCI rebuild azure-sdk-for-python

**Pull request merged**